build-tools/packages/build-cli/README.md

```diff
@@ -177,6 +179,25 @@ DESCRIPTION
   Checks that the dependencies between Fluid Framework packages are properly layered.
 ```
 
+## `flub collect bundleanalyses`
```

Comment:
nit: I think bundleAnalysis is better, even though analyses would be more correct grammatically. Actually... What do you think about bundleStats? a lot easier to type :)

Reply:
I love bundleStats more :)